qiskit/providers/ibmq/job/exceptions.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2018, 2019.
```

Comment:
Not that we have been consistent with copyrights, but it really should be `<first introduced year>, <last modified year>`, and if the 2 are the same, it'd just be 1 year. For example, since this is a new file, it should say `(C) Copyright IBM 2019.`